chart/values.yaml

```diff
@@ -1328,6 +1328,11 @@ redis:
   affinity: {}
   tolerations: []
 
+  # When not set, the values defined in the global securityContext will be used
```

Comment:
```suggestion
  uid: 999
  # If not set, `redis.uid` will be used
```

Reply:
I'll add that using `999` as the default does change the behavior as it runs as `redis` not `root`, but that worked for me even if the redis db was owned as `root`. If we want to be extra safe though, we might consider defaulting to `0` instead.

Reply:
Probably safer to default to 0 and make a note that this is for backwards-compatibility